### ISSUE TITLE
Option to delete HTML file also if validation failed

### ISSUE THREAD

**Issue by Piedone** [atata-framework/atata-htmlvalidation#3]
Since the HTML input [is currently always written to file](https://github.com/atata-framework/atata-htmlvalidation/blob/5933575f57b40196c1fdf9b3bef3038256dad4d1/src/Atata.HtmlValidation/HtmlValidator.cs#L50) I'd like to see an option to remove it also when validation fails. This [currently only happens if validation succeeds](https://github.com/atata-framework/atata-htmlvalidation/blob/5933575f57b40196c1fdf9b3bef3038256dad4d1/src/Atata.HtmlValidation/HtmlValidator.cs#L73).

**Comment by YevgeniyShunevych** [comment]
@Piedone, OK we can do that. I think of replacing `bool KeepHtmlFileWhenValid` property of `HtmlValidationOptions` with `HtmlSaveCondition SaveHtmlToFile`.

```cs
public enum HtmlSaveCondition
{
    Never,
    Invalid,
    Always
}
```

**Comment by Piedone** [comment]
That looks perfect.

**Comment by YevgeniyShunevych** [comment]
@Piedone, it is implemented in the recently released v1.1.0.

**Comment by Piedone** [comment]
Thank you, works well.
